[Update 8] Update release note with LS changes (#7588)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

### New features

#### Language Server

> - Add custom completions for http services

Reviewer: ```suggestion
- Add custom completions for HTTP services.
```

#### CLI

- Added a new `bal tool` command to manage tools that extend the functionality of the CLI. Tools can be pulled from the Ballerina Central and are managed using the `bal tool` command. For more information, see [Tool commands](https://ballerina.io/learn/cli-documentation/cli-commands/#tool-commands).

### Improvements

> #### Language Server

Reviewer: ```suggestion
#### Language Server
```
Add full stops for all the bullet points.

> - Suggest ballerina central packages when they're partially typed

Reviewer: ```suggestion
- Suggest Ballerina Central packages when they are partially typed
```

> - Improve sorting in Record type descriptor node context

Reviewer: ```suggestion
- Improve sorting in the Record-type descriptor node context.
```

> - Introduce a codeaction to add local module dependencies to Ballerina.toml

Reviewer: ```suggestion
- Introduce a code action to add local module dependencies to the `Ballerina.toml` file.
```

> - Introduce change variable type code action for let expression

Reviewer: ```suggestion
- Introduce a code action to change the variable type of a `let` expression.
```
Be consistent with the above format.

> - Introduce create function code action for the expression of select clause

Reviewer: ```suggestion
- Introduce a code action to create a function for the expression of a `select` clause.
```

> - Improve completions in Service Declaration node context

Reviewer: ```suggestion
- Improve completions in the service declaration node context.
```

> - LS simulator improvements

Reviewer: ```suggestion
- Improved the LS simulator.
```
Need to specify the improvements precisely.
